backend/src/routes/report.js

```diff
@@ -9,6 +9,24 @@ const routes = [
     schema: validators.getOneValidator,
     handler: controller.getOne,
   },
+  {
+    method: "GET",
+    url: `/${what}/projects/ProjectBudgetReport`,
+    schema: validators.getOneValidator,
+    handler: controller.getProjectBudgetReport,
+  },
+  {
+    method: "GET",
+    url: `/${what}/projects/ProjectQuarterlyReport`,
+    schema: validators.getOneValidator,
+    handler: controller.getProjectQuarterlyReport,
+  },
+  {
+    method: "GET",
+    url: `/${what}/projects/ProjectStatusReport`,
```

Comment:
Route urls should be lowercase right?

Reply:
yes i would agree.  although i'm not sure if it is case sensitive, leave no ambiguity